test/cli-commands/ganache/ganache.js

```diff
@@ -102,16 +109,20 @@ describe('Ganache cli command', () => {
 			assert.equal(tenthOutputtedAddress, TENTH_ACCOUNT_ADDRESS, 'There is mismatch of tenth account address');
 			assert.equal(tenthOutputtedPrivateKey, TENTH_PRIVATE_KEY, 'There is mismatch of tenth account private key');
 
-			killProcessByPID(childResponse.process.pid);
 		});
+		after(async () => {
```

Comment:
Like always, copy pasting of the same code is bad thing. If you put afterEach in wrapping describe, you can achieve the same thing